pallene/driver.lua

```diff
@@ -103,8 +103,8 @@ end
 -- [output_ext]. Erases any intermediate files that are produced along the way.
 --
 -- Example:
---    compile("titan", "so", "foo.titan") --> outputs "foo.so"
---    compile("titan", "c", "foo.titan")  --> outputs "foo.c"
+--    compile("titan", "so", "foo.pallene") --> outputs "foo.so"
+--    compile("titan", "c", "foo.pallene")  --> outputs "foo.c"
```

Comment:
These need to become `compile("pallene", ...` as well

Reply:
Got it, will do.